lib/src/main/java/io/ably/lib/realtime/AblyRealtimeBase.java

```diff
@@ -38,19 +47,19 @@ public class AblyRealtime extends AblyRest implements AutoCloseable {
      * @param key String key (obtained from application dashboard)
      * @throws AblyException
      */
-    public AblyRealtime(String key) throws AblyException {
-        this(new ClientOptions(key));
+    public AblyRealtimeBase(String key, PlatformAgentProvider platformAgentProvider) throws AblyException {
```

Comment:
PlatformAgentProvider is not documented

Reply:
https://github.com/ably/ably-java/pull/768/commits/fc6933df622857f7fa830cf253d5d43f5b61ce81